src/Collector/Profile.php

```diff
@@ -13,32 +13,14 @@
  */
 final class Profile
 {
-    /**
-     * @var string
-     */
-    private $plugin;
+    private string $request;
```

Comment:
this changes the behaviour: if we try to getRequest before a request is set, php will do a fatal error because an unitialized property is accessed.

i think we should initialize with null to keep the rendering of the profile the same as before:

```suggestion
    private ?string $request = null;
```

Reply:
++